data/oyepriyansh.json

```
@@ -0,0 +1,238 @@
+{
+
+	"name": "Priyansh Prajapat",
+
+	"type": "personal",
+
+	"bio": "Aspirant Full stack Web Developer",
+
+
+
+	"socials": [
+
+		{
+			"icon": "FaInstagram",
+			"url": "https://instagram.com/oyepriyansh"
+		},
+
+		{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/oyepriyansh"
+		},
+
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/oyepriyansh"
+		}
+
+	],
+
+
+
+	"links": [
+
+		{
+
+			"group": "Overview",
+
+			"name": "Portfolio",
+
+			"url": "https://oyepriyansh.pages.dev",
+
+			"icon": "FaGlobe"
+
+		},
+
+		{
+
+			"group": "Overview",
+
+			"name": "Minifolio",
+
+			"url": "https://oyepriyansh.github.io",
+
+			"icon": "FaLink"
+
+		},
+
+		{
+
+			"group": "Overview",
+
+			"name": "Email",
+
+			"url": "mail to:hi@priyansh.app",
```

There should be no space in between a URL. Rest looks good to me! 

```suggestion
			"url": "mailto:hi@priyansh.app",
```